include/mata/nfa/nfa.hh

```diff
@@ -252,7 +252,8 @@ public:
      *
      * The operation has constant time complexity.
      */
-    size_t get_num_of_trans() const { return static_cast<size_t>(std::distance(delta.begin(), delta.end())); }
+    size_t get_num_of_trans() const { return static_cast<size_t>(std::distance(delta.transitions_begin(),
```

Comment:
can it now be Transitions.size()?

Reply:
Yes. It can. I will refactor this.

Reply:
ok

Reply:
There will be a separate PR on adding `Transtiions` as a member of `Delta`.

Reply:
Transitions::count() added.